cmd/argot/main.go

```diff
@@ -43,7 +43,7 @@ func main() {
 	incluster := couple.Flag("incluster", "use in-cluster configuration.").Bool()
 	kubeconfig := couple.Flag("kubeconfig", "path to kubeconfig (if not in running inside a cluster)").Default(filepath.Join(os.Getenv("HOME"), ".kube", "config")).String()
 	ingressclass := couple.Flag("ingress-class", "ingress class name").Default(argotunnel.IngressClassDefault).String()
-	originsecret := k8s.ObjMixin(couple.Flag("default-origin-secret", "default origin certificate secret <namespace>/<name>"))
+	originsecrets := k8s.ObjMixin(couple.Flag("default-origin-secret", "default origin certificate secret <namespace>/<name>"))
```

Comment:
this changes the semantic meaning of default, I wouldn't want to change the meaning of a parameter without at least a minor version change (thats why I waited till 0.6.x to introduce these and alter the old ones).